spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+feature 'user signs in', %Q{
+As an unauthenticated user
+I want to be able to sign in
+So that I can gain access to the app
+} do
+
+# Acceptance Criteria:
+# [X] There is an option to sign in if I am not signed in
+# [X] My password is not displayed as I enter it
+# [X] If I've entered the right credentials, I receive an onscreen message that I'm signed in, and I've gained access to the system
+# [X] If I'm already authenticated, I don't have the option to sign in
+# [X] If I didn't enter valid credentials, I receive an error message and I am left unauthenticated
```

Comment:
Line is too long. [99/80]<br>Incorrect indentation detected (column 0 instead of 2).